defaults/main.yml

```diff
@@ -39,6 +44,10 @@ go_waku_metrics_enabled: true
 go_waku_metrics_port: 8008
 go_waku_websocket_enabled: false
 go_waku_websocket_port: 9001
+go_waku_websockets_secure_enabled: false
+go_waku_websocket_ssl_dir: ''
+go_waku_websocket_ssl_cert: ''
+go_waku_websocket_ssl_key: ''
```

Comment:
These should be commented out, and with examples, like in `nim-waku` role:
https://github.com/status-im/infra-role-nim-waku/blob/781a3038f1a664f2645d10373016e421ca49d386/defaults/main.yml#L67-L73